docs/building/ubuntu-instructions.md

```diff
@@ -179,23 +179,23 @@ Once you build the samples, you can use `spark-submit` to submit your .NET Core
      ```
```

Comment:
@bamurtaugh Could you divide this into multiple smaller PRs to make reviews easier? You can have one PR for restructuring, and one PR for each sample. Thanks!

Reply:
@imback82 Do you have any recommendations for how to do that? Would I close this PR and individually open up new branches/thus new PRs with subsets of this overall list of commits?

Reply:
Yes, one PR for each sample. You can leave this PR open until all subset PRs are merged. Thanks! (and this will expedite the review process as well)

Reply:
Opened new PRs (one for each sample, and an additional PR for some readme additions/updates). Thank you!